agent/engine/types.go

```diff
@@ -38,3 +38,10 @@ type DockerContainerMetadata struct {
 	Error        error
 	Volumes      map[string]string
 }
+
+// ListContainersResponse encapsulates the response from the docker client for the
+// ListContainers call.
+type ListContainersResponse struct {
+	ContainerIds []string
```

Comment:
In other places this is called "DockerId", for consistency it would be nice to have this as 'DockerIds'.